macros/src/actix/mod.rs

```diff
@@ -179,31 +213,7 @@ fn handle_field_struct(fields: &FieldsNamed, props_gen: &mut proc_macro2::TokenS
             .to_string();
 
         let mut is_required = true;
-        let ty_ref = match field.ty {
-            Type::Path(ref p) => {
-                let ty = p
-                    .path
-                    .segments
-                    .last()
-                    .expect("expected type for struct field");
-
-                if p.path.segments.len() == 1 && &ty.ident == "Option" {
-                    is_required = false;
-                }
-
-                address_type_for_fn_call(&field.ty)
-            }
-            Type::Reference(_) => address_type_for_fn_call(&field.ty),
-            _ => {
-                field
-                    .ty
-                    .span()
-                    .unwrap()
-                    .warning("unsupported field type will be ignored.")
-                    .emit();
```

Comment:
This warning should still be emitted, so that users can see that it's something we don't support now.